automation_testing/pages/DataViewPage.ts

```diff
@@ -143,4 +147,76 @@ export class DataviewPage {
 		}
 		return createdDates;
 	}
+
+	async validateContainsKeyword(titles: string[], keyword: string): Promise<void> {
+		for (const title of titles) {
+			expect(title.toLowerCase()).toContain(keyword.toLowerCase());
+		}
+	}
+
+	async validateContainsMoreThanOneKeyword(titles: string[], keywords: string[]): Promise<void> {
+
+
+		for (const title of titles) {
+			let isContaining = false;
+			for (const keyword of keywords) {
+				if (title.toLowerCase().includes(keyword.toLowerCase())) {
```

Comment:
Please pull from develop and push again because this seems to have old code that was already changed on another branch / PR.